README.md

```diff
@@ -14,11 +14,11 @@ Artemis
 
 > Artemis is the greek goddess of hunting and the moon, born of Zeus and Leto, twin of Apollo.
```

Comment:
```suggestion
```

Reply:
Actually, I think we should start with some paragraph stating the overall purpose of Wynntils. Something like:

Wynntils is a Minecraft mod for the MMORP [Wynncraft](https:///wynncraft.com). It is intended to enhance your gaming experience when playing Wynncraft.

Reply:
Or maybe reuse the wording from legacy: "Wynntils is a Wynncraft mod that seeks to enhance the user's gameplay with a variety of customizable options and additions."

Maybe we can also replace the Artemis image with this from the Legacy readme:

![logo](https://camo.githubusercontent.com/b7f2c73e3dadf738c755e8bb01bf2c5460be9641a4612f1f0b98dd85db352c34/687474703a2f2f63646e2e77796e6e74696c732e636f6d2f77796e6e74696c732532306c6f676f2532302d253230772532307061696e747374726f6b652e706e67)

Reply:
... perhaps after scaling it down a bit :)